packages/flutter/lib/src/material/toggleable.dart

```diff
@@ -330,6 +333,23 @@ abstract class RenderToggleable extends RenderConstrainedBox {
     markNeedsPaint();
   }
 
+  /// The color that should be used for the reaction when the toggleable is
+  /// inactive.
+  ///
+  /// Used when the toggleable needs to change the reaction color/transparency
+  /// that is displayed when the toggleable is inactive and tapped.
+  ///
+  /// Defaults to the [activeColor] at alpha [kRadialReactionAlpha].
```

Comment:
Should this default to inactiveColor instead?

Reply:
Same comment as above: It would break internal tests.